Coverage/MergeRunner.h

```diff
@@ -3,7 +3,7 @@
 #include "RuntimeOptions.h"
 
 #include <assert.h>
-#include <experimental/filesystem> // C++ 17  feature
+#include <filesystem>
```

Comment:
You're absolutely right that we shouldn't use experimental anymore. Not sure why it's still there...